lwgenerate/gcp/helpers.go

```diff
@@ -0,0 +1,50 @@
+package gcp
+
```

Comment:
rename this file from 'helpers.go' to something more appropriate. Or move the functions.